extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/theta/SketchAggregator.java

```diff
@@ -112,7 +112,7 @@ static void updateUnion(Union union, Object update)
     } else if (update instanceof byte[]) {
       union.update((byte[]) update);
     } else if (update instanceof Double) {
-      union.update(((Double) update));
+      union.update((Double) update);
     } else if (update instanceof Integer || update instanceof Long) {
       union.update(((Number) update).longValue());
```

Comment:
Why not also update this on this line?

Reply:
Hum, do you mean line 117 ?
I could not find redundant in this line, could you show me the example?
Sorry for that i am not so familiar with Java

Reply:
You are correct, there are no extra parenthesis here. Sorry about that.

CI is complaining because of missing tests for this Aggregator. Do you think you could take a stab at adding some unit tests for this class?

https://github.com/apache/druid/blob/master/dev/intellij-setup.md#Set-Code-Coverage-Runner documents how to run code coverage locally, so you can see which parts of the class are being tested. The unit tests appear to be in `SketchAggregationTest`

Reply:
Ok， i would work around it soon.
Thanks for your review very much.
After finishing it, i would push a commit including the updated test and reply on this PR.
Thanks again.